dev/com.ibm.ws.st.jee.core/src/com/ibm/ws/st/jee/core/internal/JEEPublisher.java

```diff
@@ -6,7 +6,7 @@
  * http://www.eclipse.org/legal/epl-v10.html
  *
  * Contributors:
- *     IBM Corporation - initial API and implementation
+ * IBM Corporation - initial API and implementation
```

Comment:
It seems Eclipse automatically reformats the license header in files when they're saved.